crates/edr_eth/src/remote/client.rs

```diff
@@ -1203,7 +1206,7 @@ mod tests {
 
         if let RpcClientError::HttpStatus(error) = error {
             assert_eq!(
-                error.status(),
+                <ReqwestError as Into<reqwest::Error>>::into(error).status(),
```

Comment:
Why did `reqwest::Error::from(..)` not work here?

Reply:
Yeah that's cleaner thanks, fixed in [Simplify error conversion](https://github.com/NomicFoundation/hardhat/pull/5067/commits/fa0abb8e7574ecdeae2c4a4a6d6761d27ac52023)